src/Modules/OrchardCore.Commerce/Drivers/PriceDisplaySettingsDisplayDriver.cs

```diff
@@ -55,7 +56,7 @@ public override async Task<IDisplayResult> EditAsync(PriceDisplaySettings sectio
             .OnGroup(GroupId);
```

Comment:
You missed this one.
```suggestion
            .OnGroup(SettingsGroupId);
```